api-report/shared-object-base.api.md

```diff
@@ -109,6 +110,8 @@ export abstract class SharedObjectCore<TEvent extends ISharedObjectEvents = ISha
     protected didAttach(): void;
     protected dirty(): void;
     // (undocumented)
+    emit(event: EventEmitterEventType, ...args: any[]): boolean;
```

Comment:
protected?

Reply:
ahh i see. this was technically already public due to the inheritance chain. i'm still curious if type script will let you mark it protected

Reply:
Nope, it complains about incorrectly extended base class.